plugins/input-exercise-renderer/src/index.ts

```diff
@@ -9,7 +9,7 @@ export const inputExerciseRendererPlugin: RendererPlugin<
 }
 
 export interface InputExercisePluginState {
-  correctValue: string
+  correctAnswers: CorrectAnswer[]
```

Comment:
Does this need to be an object?

Reply:
no this does not need to be an object.  I made it an object in anticipation of a need to allow  feedback for various accepted answers. If we revert to string there will have to be a number of alterations throughout plugin

Reply:
Fair enough 👍